File: Modul5/Python/PRAK505-2310817210003-MuhammadAzrianzan.py
def Biodata(tahunLhr, nama, kota):
    tahun_sekarang = 2023
    print(f"\nPerkenalkan Nama Saya: {nama}")
    print(f"Umur Saya: {tahun_sekarang - tahunLhr}")
    print(f"Saya Adalah Angkatan: {tahun_sekarang}")
    print(f"Asal Saya dari: {kota}")

tahunLahir = int(input())
Namaku = str(input())
Asal = str(input())
Biodata(tahunLahir, Namaku, Asal)

File: Modul5/C/PRAK505-2310817210003-MuhammadAzrianzan.c
#include <stdio.h>

void Biodata(int tahunLhr, char *nama, char *kota){
    int tahun_sekarang = 2023;
    printf("\nPerkenalkan Nama Saya: %s\n", nama);
    printf("Umur Saya: %d\n", tahun_sekarang-tahunLhr);
    printf("Saya Adalah Angkatan: %d\n", tahun_sekarang);
    printf("Asal Saya dari: %s\n", kota);
}

int main() {
    int tahunLahir;
    char Namaku[20], Asal[15];
    scanf(" %d",&tahunLahir);
    scanf(" %[^\n]%*c",&Namaku);
    scanf(" %[^\n]%*c",&Asal);
    Biodata(tahunLahir, Namaku, Asal);
    return 0;
}
